tests/integration/components/flexberry-field-test.js

```diff
@@ -54,18 +75,18 @@ test('label mode works properly', function(assert) {
 
   // Check that label attribute doesn't exist now.
   this.set('label', null);
-  assert.strictEqual(this.get('label'), null, 'Component\'s hasn\'t wrapper is a <lable>');
+  assert.strictEqual(this.get('label'), null, 'Component\'s hasn\'t wrapper is a <label>');
 
   // Add text for label & check that label attribute exist.
   let labelText = 'Some text for label';
   this.set('label', labelText);
 
   assert.strictEqual(
-  this.get('label'), labelText, 'Component\'s has wrapper is a <lable>');
+  this.get('label'), labelText, 'Component\'s has wrapper is a <label>');
 
   // Check that label attribute doesn't exist now.
   this.set('label', null);
-  assert.strictEqual(this.get('label'), null, 'Component\'s hasn\'t wrapper is a <lable>');
+  assert.strictEqual(this.get('label'), null, 'Component\'s hasn\'t wrapper is a <label>');
```

Comment:
Typo fixed, but assert comment is still strange. More siutable is "Component hasn't inner `<label>`".